server.py

```diff
@@ -109,11 +109,33 @@ def __init__(self, map_name):
                     await client.send_json(self.state.as_dict(map_name))
             return ws
 
+
+    async def play_round(self, state):
+        """
+        If all robot have selected cars, server apply effects of cards and tiles.
+        New dealt cards are sent to all clients.
+        """
+        all_selected = True
+        for robot in self.state.robots:
```

Comment:
Blok kodu v tomto for cyklu funguje jen jednou pri prvnim kole hry, pak uz ne. Robot ma nastaven atribut "selection_confirmed", ale nikdy se mu tento atribut "neodnastavi" - asi nekde v bloku po provedeni efektu hry.

Reply:
doplněno, nastaví se zpět podle příchozí zprávy na začátku